src/ui/profile_preferences.ui

```diff
@@ -2665,6 +2665,20 @@ you can use it but there could be issues with aligning columns of text</string>
             </property>
            </widget>
           </item>
+          <item row="1" column="2">
```

Comment:
Can you hand edit this (use the "plain text editor" if working in Qt Creator) and put these `<item ...>`s in the right place in this `.ui` file?

ℹ️ You can fold up the members of this `QGridLayout` (click on the `-` (shown) / `+` (hidden) symbols in the left margin) in that editor and then cut and paste the single line collapsed form in the right place...